src/drafts/SelectPanel2/SelectPanel.stories.tsx

```diff
@@ -773,6 +773,64 @@ export const GOpenFromMenu = () => {
   )
 }
 
+export const IWithRemoveFilterIcon = () => {
```

Comment:
Do we want to keep this story?

Reply:
I believe so? If only because it will be easy to take out later if we want 🙇🏻